careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
+**Impact:** They can take the lead on delivering work critical to meeting the sprint goal and are capable of independently planning work that multiple engineers can complete in a sprint or two. Instead of just executing on a task as written, they think critically about the problem and ensure that the solution will thoroughly solve the problem.
 
-Engineer IIs can take a bug or a task, and instead of just executing on the task as written, think critically about the problem and ensure that the issue has been thoroughly resolved by identifying root causes, proactively addressing performance concerns, providing proactive feedback on any product requirements and writing solid tests.
+**Communication:** They are able to effectively review code in their area of expertise and regularly point out opportunities for improvement in the code that they review. They proactively point out concerns with proposed or current sprint work as soon as they see them. They can articulate the tradeoffs of different approaches to solving a problem.
 
-By the time an engineer is ready to be promoted to Engineer III they will have focused on a technology or functional area as their expertise and become capable of mentoring interns and new engineers in these areas.
+**Collaboration:** They are mindful of the work that others are doing and can critique or suggest improvements to the proposed plans of others. They get help from others when stuck, but are also capable of helping unstick their peers. They are capable of effectively running small to medium-sized projects with regular coaching.
 
 ## Engineer III
 
-**Technical Ability:** An Engineer III can consistently be relied on to take on an undefined project through to execution while fully and effectively utilizing the skills and intelligence of the team. An Engineer III has the ability to provide an answer to most technical questions that the team may have, but they make a choice to exercise restraint and ask questions to help other team members reach their own conclusions.
+An engineer III is capable of taking on complex technical problems and coordinating work spanning several months.
 
-**Technical Scope:** An Engineer III takes ownership over the "how" of the feature they implement with a "buck stops here" attitude. If an issue/bug does arise, they see developing a solution to that problem as their personal responsibility. An Engineer III actively seeks out knowledge about the risks and challenges in our business, using that knowledge to guide decision-making
+**Technical Knowledge:** They have a robust understanding of their technical specialty, and a working knowledge of other domains. They can solve most problems in their domain. Their code is well-abstracted and performant as a rule and rarely needs a significant refactor at the PR stage. If an unexpected issue/bug does arise, they see developing a solution to that problem as their personal responsibility.
```

Comment:
I think you can just drop "as a rule" since you follow it up with noting it rarely needs a significant refactor (for any reason, abstraction or performance aside).